src/test/java/de/tum/in/www1/artemis/Lti13LaunchIntegrationTest.java

```diff
@@ -74,7 +72,7 @@ void redirectProxy() throws Exception {
         body.put("id_token", VALID_ID_TOKEN);
         body.put("state", VALID_STATE);
 
-        URI header = request.postForm(CustomLti13Configurer.LTI13_LOGIN_REDIRECT_PROXY_PATH, body, HttpStatus.FOUND);
+        URI header = request.postForm("/api/public/lti13/auth-callback", body, HttpStatus.FOUND);
```

Comment:
(Just curious no need to change) but why did you replace the constants here?

Reply:
The constants are used in the API definition and are therefore validated by my architectute test. Since the constants started with /api, the test failed. So I removed the slash and added it back in the places where it is necessary, so above and in the tests. For the tests I find it easier to better to test an endpoint literal, that's why I inlined them here.